crates/fluvio-hub-util/src/cmd/list.rs

```diff
@@ -88,15 +104,54 @@ mod output {
 
         /// table content implementation
         fn content(&self) -> Vec<Row> {
-            self.0
+            self.pkgs
                 .iter()
                 .map(|e| {
                     Row::from([
                         Cell::new(e.pkg_name()).set_alignment(CellAlignment::Left),
                         Cell::new(&e.visibility).set_alignment(CellAlignment::Left),
+                        Cell::new(
+                            e.published_at()
+                                .map(|date| format_duration(date, self.exact_time))
+                                .unwrap_or(String::from("N/A")),
+                        ),
                     ])
                 })
                 .collect()
         }
     }
 }
+
+fn format_duration(date: DateTime<Utc>, exact: bool) -> String {
+    let duration = Utc::now().signed_duration_since(date);
+
+    if exact {
+        return date.format("%Y-%m-%d %H:%M:%S %Z").to_string();
+    }
+
+    if duration.num_weeks() >= 1 {
```

Comment:
This ia already available in humantime crate.  No need to reinvent

Reply:
Yes that was my first attempt, using humantime, but I didn't found a format specifier,[`FormatDuration`](https://docs.rs/humantime/latest/humantime/fn.format_duration.html) doesn't let you pick which fields to use.

And given that release dates are from more that a year in some cases we have outputs like `2years 2min 12us` (as shown here: https://docs.rs/humantime/latest/humantime/)

Reply:
Here is an example of how many packages looked like: `2years 12h 4m 37s 47ms 997us 519ns`

https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=933296e2dc7dcd9f93dc91fc75849617

It would be nice to use from `years` to `minutes` but thing is that theres no way to specify that.

Reply:
That's too much. It should default to the largest value `2 years`, or `4 days`, or `26 min`. 
Maybe anything in hours would show `3 hours 4 min`, but nothing more granular than that.

Reply:
BTW, you can use Kubernetes pods output for hints.

Reply:
Thats correct @ajhunyady! Thank you so much for the suggestions!

I have updated tests to use a static past date and use that for calculations, so dynamic current time doest break them.

Yeah it would be nice to have humantime format support but seems that its not being actively maintained, last commit was 2 years ago: https://github.com/tailhook/humantime